reference/src/client/test/sa_crypto_cipher_process_rsa_pkcs1v15.cpp

```diff
@@ -133,7 +133,9 @@ namespace {
 
         ASSERT_EQ(status, SA_STATUS_OK);
         ASSERT_NE(cipher, nullptr);
-
+        
```

Comment:
This should be removed and submitted as a separate PR tied to the Resolve OpenSSL 3.x.x unit test discrepancies issue since this is not functionality related to TA key provisioning.